### PR TITLE
Give workflow a name

### PR DESCRIPTION
**Pull request by gewarren**
Give workflow a name so it doesn't appear.

Old look:

image

New look:

image

### CODE REVIEW

File: .github/workflows/check-for-build-warnings.yml [hunk 1]
```diff
@@ -10,8 +12,8 @@ on:
```
Comment (gewarren) [inline .github/workflows/check-for-build-warnings.yml:12]: @BillWagner For the manual run to be useful, we'd have to pass in a PR number. Do you want me to change the action to take a PR number as input?

https://github.com/dotnet/docs/runs/2273417489?check_suite_focus=true

Comment (BillWagner) [reply .github/workflows/check-for-build-warnings.yml:12]: Ping @IEvangelist who taught me this trick on the manual run.

**Review by BillWagner — APPROVED** [review]
This LGTM @gewarren 

You can :shipit: when ready.

File: .github/workflows/check-for-build-warnings.yml (hunk 1 above)
Comment (IEvangelist) [reply .github/workflows/check-for-build-warnings.yml:12]: @gewarren yes, you'd have to update the action to take an optional PR number. Then you'd essentially [coalesce on the payload](https://github.com/dotnet/samples/blob/main/.github/actions/status-checker/src/status-checker.ts#L10), as that would be `null` when manually ran.

Comment (gewarren) [reply .github/workflows/check-for-build-warnings.yml:6]: Thanks, I think it's best to just remove the manual trigger for now. We can add it later if we decide it's needed.

**Pull request merged**